src/pages/StatusPage.vue

```diff
@@ -54,6 +54,12 @@
                     <label class="form-check-label" for="show-powered-by">{{ $t("Show Powered By") }}</label>
                 </div>
 
+                <!-- Show certificate expiry -->
+                <div class="my-3 form-check form-switch">
+                    <input id="show-certificate-expiry" v-model="config.showCertificateExpiry" class="form-check-input" type="checkbox">
+                    <label class="form-check-label" for="show-certificate-expiry">{{ $t("Show Certificate Expiry") }}</label>
```

Comment:
Could you add this key `Show Certificate Expiry` to `en.json` (and any languages for which you may be a native speaker)?
Otherwise, translators won't be able to translate this. ^^